.github/workflows/test.yml

```
@@ -24,18 +24,20 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node: [14, 16]
+        node: [18]
```

```suggestion
        node: [18, 20, 22]
```
I hope Node stable version!